fdbserver/TLogServer.actor.cpp

```diff
@@ -2813,6 +2830,15 @@ ACTOR Future<Void> serveTLogInterface(TLogData* self,
 			} else {
 				logData->logSystem->set(Reference<ILogSystem>());
 			}
+			if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST && logData->stopped()) {
+				for (auto& oldTLogs : self->dbInfo->get().logSystemConfig.oldTLogs) {
+					if (oldTLogs.epoch >= logData->epoch()) {
```

Comment:
why is the comparison not `==`? E.g., if the latest old epoch is 14, and the logData is at epoch 10, then this doesn't seems right.

Reply:
I added a note in the commit message a note on this. "The old tLog list in dbinfo may not contain the same epoch as the recovering tLog. If this happens the first entry in the list with an epoch greater than the tLog's epoch will contain the recovery version." If the epoch 10 is not part of the oldTLog list, it could not have received any commits, so it is safe to use the subsequent epoch.